gourmet/importers/plaintext_importer.py

```diff
@@ -45,32 +45,32 @@ def handle_line (self):
         raise NotImplementedError
 
     def compile_regexps (self):
-        self.blank_matcher = re.compile("^\s*$")
+        self.blank_matcher = re.compile(r"^\s*$")
         # out unwrap regexp looks for a line with no meaningful characters, or a line that starts in
         # ALLCAPS or a line that is only space. (we use this with .split() to break text up into
         # paragraph breaks.
-        self.unwrap_matcher = re.compile('\n\W*\n')
-        self.find_header_breaks_matcher = re.compile('\s+(?=[A-Z][A-Z][A-Z]+:.*)')
+        self.unwrap_matcher = re.compile(r'\n\W*\n')
+        self.find_header_breaks_matcher = re.compile(r'\s+(?=[A-Z][A-Z][A-Z]+:.*)')
 
     def unwrap_lines (self, blob):
         if blob.find("") >= 0:
             debug('Using built-in paragraph markers',1)
             # then we have paragraph markers in the text already
-            outblob = string.join(blob.split("\n")," ") # get rid of line breaks
+            outblob = " ".join(blob.split("\n")) # get rid of line breaks
             lines = outblob.split("") # split text up into paragraphs
```

Comment:
Can we replace this with `splitlines()` as we're here? https://docs.python.org/3/library/stdtypes.html#str.splitlines